assets/html/scss/documenter/layout/_main.scss

```
@@ -73,14 +73,6 @@
           padding: 0;
           margin-left: 0.3em;
         }
-        .docs-navbar-link {
-          margin: auto 0.5rem auto 0.5rem;
-        }
-        @include touch {
-          .docs-navbar-link {
-            margin: auto 0.75rem auto 0.75rem;
-          }
-        }
       }
```

Maybe let's keep the touch margins, but make them a bit smaller? Point being, it's good to have a bit of space around the buttons when hitting them with your finger on a mobile screen.

```suggestion
        @include touch {
          .docs-navbar-link {
            margin-left: 0.4rem;
            margin-right: 0.4rem;
          }
        }
      }
```